app/controllers/hq/users_controller.rb

```diff
@@ -6,4 +6,24 @@ def index
     @users = User.all
   end
 
+  def edit
+    @user = User.find(params[:id])
```

Comment:
Unless there are well-founded objections, let's stick to the example set by Partners controller with `load_user` etc.?